prov/gni/src/gnix_ep.c

```diff
@@ -474,6 +475,65 @@ static ssize_t gnix_ep_recvmsg(struct fid_ep *ep, const struct fi_msg *msg,
 
 }
 
+int _gnix_send_req(void *data)
+{
+	struct gnix_fab_req *req = (struct gnix_fab_req *)data;
+	struct gnix_nic *nic;
+	struct gnix_fid_ep *ep;
+	struct gnix_tx_descriptor *tdesc;
+	gni_return_t status;
+	int ret;
+
+	ep = req->gnix_ep;
+	assert(ep != NULL);
+
+	nic = ep->nic;
+	assert(nic != NULL);
+
+	if (req->len < (16384 - sizeof(struct gnix_smsg_hdr))) {
```

Comment:
Noted, but what is 16384 supposed to be? This should have been a constant defined somewhere that gave some context perhaps. I assume the largest smsg that can be sent is 16384?